qa/setup_packages.py

```diff
@@ -438,7 +438,7 @@ def get_pyvers_name(self, url, cuda_version):
                           "110" : [
                               PckgVer("1.15.5",  python_max_ver="3.7"),
                               "2.5.1",
-                              PckgVer("2.6.0", dependencies=["tensorflow-estimator==2.6.0"]),
+                              PckgVer("2.6.0", dependencies=["tensorflow-estimator==2.6.0", "keras==2.6.0"]),
```

Comment:
It's weird that we don't have to pin it for other versions.

Reply:
It is. I bet it is about TF itself not pinning the dependency version in the case of 2.6.x.

Reply:
It is. I bet it is TF that doesn't pin its dependencies for 2.6.x.